Mage/src/main/java/mage/game/permanent/token/BlackBirdToken.java

```diff
@@ -0,0 +1,35 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.common.CantBlockAbility;
+import mage.abilities.keyword.FlyingAbility;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+/**
+ *
+ * @author weirddan455
+ */
+public final class BlackBirdToken extends TokenImpl {
```

Comment:
Should the `availableImageSetCodes` be set later after the token image is available on scryfall, or can it be set now without issue?

Reply:
I don't know how that works exactly.  @JayDi85 usually does that after we're finished programming the set.